### ISSUE TITLE
NullPointer in the ubuntu

### ISSUE THREAD

**Issue by RunningForever** [AlmasB/FXGL#307]
I use IDEA in the ubuntu
I add 'compile 'com.github.almasb:fxgl:0.3.1' in build.gradle
and then add jcenter() in repositories
The BaseGameAPP is:
```java
import com.almasb.fxgl.app.GameApplication;
import com.almasb.fxgl.entity.Entities;
import com.almasb.fxgl.entity.GameEntity;
import com.almasb.fxgl.input.UserAction;
import com.almasb.fxgl.service.Input;
import com.almasb.fxgl.settings.GameSettings;
import javafx.scene.input.KeyCode;
import javafx.scene.paint.Color;
import javafx.scene.shape.Rectangle;
import javafx.scene.text.Text;

import java.util.Map;

public class BaseGameApp extends GameApplication {
    private GameEntity player;

    @Override
    protected void initSettings(GameSettings settings) {
        settings.setWidth(600);
        settings.setHeight(600);
        settings.setTitle("Basic Game App");
        settings.setVersion("0.1");
        settings.setProfilingEnabled(false);
        settings.setCloseConfirmation(false);
        settings.setIntroEnabled(false);
    }

    @Override
    protected void initGame() {
        player = Entities.builder()
                .at(300, 300)
                .viewFromNode(new Rectangle(25, 25, Color.BLUE))
                .buildAndAttach(getGameWorld());
    }

    @Override
    protected void initInput() {
        Input input = getInput(); // get input service

        input.addAction(new UserAction("Move Right") {
            @Override
            protected void onAction() {
                player.translateX(5); // move right 5 pixels
            }
        }, KeyCode.D);

        input.addAction(new UserAction("Move Left") {
            @Override
            protected void onAction() {
                player.translateX(-5); // move left 5 pixels
            }
        }, KeyCode.A);

        input.addAction(new UserAction("Move Up") {
            @Override
            protected void onAction() {
                player.translateY(-5); // move up 5 pixels
            }
        }, KeyCode.W);

        input.addAction(new UserAction("Move Down") {
            @Override
            protected void onAction() {
                player.translateY(5); // move down 5 pixels
            }
        }, KeyCode.S);
    }

    @Override
    protected void initUI() {
        Text textPixels = new Text();
        textPixels.setTranslateX(50); // x = 50
        textPixels.setTranslateY(100); // y = 100

        getGameScene().addUINode(textPixels); // add to the scene graph
    }

    @Override
    protected void initGameVars(Map<String, Object> vars) {
        vars.put("pixelsMoved", 0);
    }
}

```

then Run the BaseGameAPP.main()
and when i click new Game option,the error occured
the log is:
```
INFO: FXGL-0.3.1
INFO: Source code and latest versions at: https://github.com/AlmasB/FXGL
INFO:              Join the FXGL chat at: https://gitter.im/AlmasB/FXGL
08:38:22.891 [FXGL Launcher Thread     ] INFO  FXGLApplication      - FXGL configuration took:  1.074 sec
08:38:23.285 [FXGL Launcher Thread     ] INFO  FXGLApplication      - Game configuration took:  0.378 sec
08:38:25.187 [JavaFX Application Thread] WARN  IOTask               - Task failed: loadFileNamesTask(./profiles/adas/saves/, true, [FileExtension(extension=.sav)]) Error: java.io.FileNotFoundException: No save files found
08:38:58.176 [JavaFX Application Thread] FATAL FXGLExceptionHandler - Uncaught Exception:
08:38:58.177 [JavaFX Application Thread] FATAL FXGLExceptionHandler - 

Exception occurred: java.lang.NullPointerException : null
E: com.almasb.fxgl.app.InitAppTask.scanForAnnotations(InitAppTask.kt:123)
E: com.almasb.fxgl.app.InitAppTask.call(InitAppTask.kt:73)
E: com.almasb.fxgl.app.InitAppTask.call(InitAppTask.kt:45)
E: javafx.concurrent.Task$TaskCallable.call(Task.java:1423)
E: java.util.concurrent.FutureTask.run(FutureTask.java:266)
E: java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
E: java.util.concurrent.FutureTask.run(FutureTask.java:266)
E: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
E: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
E: java.lang.Thread.run(Thread.java:745)

08:38:58.177 [JavaFX Application Thread] FATAL FXGLExceptionHandler - Application will now exit
```
I don't know how to solve it

**Comment by AlmasB** [comment]
Thanks for running this. I've found the bug and it will be fixed in the next version. To make it work with `0.3.1` just add a package to your class, e.g. `package tutorial;`.

**Comment by AlmasB** [comment]
Fixed in `0.3.2`
